config/kibana.yml

```diff
@@ -61,6 +61,11 @@
 # must be a positive integer.
 # elasticsearch.requestTimeout: 30000
 
+# List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
+# headers, set this value to null, not []. Setting it to [] will send the default list
```

Comment:
So I tried to make it so `[]` sends no headers but I couldn't get that working with Joi. I think the issue is that it treats `[]` as empty and then replaces it with the default, which is `[ "authorization" ]`. That's why I had to play this silly game with null, which I don't like either.